cms/envs/common.py

```diff
@@ -1216,8 +1219,6 @@
 # 5 minute expiration time for JWT id tokens issued for external API requests.
 OAUTH_ID_TOKEN_EXPIRATION = 5 * 60
 
-USERNAME_PATTERN = r'(?P<username>[\w.@+-]+)'
```

Comment:
Why we delete this line? In `common.py` should be default values and after this in `aws.py` or `production.py` we can override them from the `*.envs.json` config files.

Reply:
It is imported on the line  84 from lms/common.py file. It was copied from hawthorn release and it looks logical, because the same pattern is not duplicated

Reply:
And the pattern will be the same for the whole system